components/filter/filter.js

```diff
@@ -78,34 +86,52 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 	}
 
 	render() {
-		const header = this._buildHeader();
-		const dimensions = this._buildDimensions();
+		const singleDimension = this._dimensions.length === 1;
+		const header = this._buildHeader(singleDimension);
+		const dimensions = this._buildDimensions(singleDimension);
 
-		return html`
-			<d2l-dropdown-button-subtle
-				@d2l-dropdown-close="${this._handleDropdownClose}"
-				@d2l-dropdown-open="${this._stopPropagation}"
-				@d2l-dropdown-position="${this._stopPropagation}"
-				text="${this.localize('components.filter.filters')}"
-				?disabled="${this.disabled}">
+		const buttonText = singleDimension ? this._dimensions[0].text : this.localize('components.filter.filters');
+		const description = singleDimension ? this.localize('components.filter.singleDimensionDescription', 'filterName', this._dimensions[0].text) : this.localize('components.filter.filters');
```

Comment:
These will eventually also incorporate the counts

Reply:
This way is still supported because of Polymer, but I think we prefer the `{ filterName: this._dimensions[0].text }` way.